browser/caliban.py

```diff
@@ -601,14 +602,13 @@ def action_predict_zstack(self):
         self.create_cell_info(feature = self.feature)
 
     def action_save_zstack(self):
-        save_file = self.filename + "_save_version_{}.npz".format(self.save_version)
+        # save file to BytesIO object
+        store_npz = BytesIO()
+        np.savez(store_npz, raw = self.raw, annotated = self.annotated)
```

Comment:
Since we're changing the line, let's lint it up a bit.

Spaces go around the `=` operator UNLESS the it is inside a function call, assigning named arguments.

This line can just be `np.savez(store_npz, raw=self.raw, annotated=self.annotated)`